src/react/notification-banner/__examples__/success.tsx

```diff
@@ -0,0 +1,13 @@
+import { NotificationBanner } from '../NotificationBanner'
+
+export const Example = () => (
+  <NotificationBanner type='success'>
+    <h3 className='govuk-notification-banner__heading'>
+      Training outcome recorded and trainee withdrawn
+    </h3>
+    <p className='govuk-body'>
+      Contact <a className='govuk-notification-banner__link' href='#'>example@department.gov.uk</a>
+      {' if you think there’s a problem.'}
```

Comment:
With `{' '}if you think there’s a problem.` React inserts an HTML comment, which affects rendering in Chrome for some reason. So `{' if you think there’s a problem.'}` is used instead (similar for the with-html example).